src/lib/utility/calendar.js

```diff
@@ -454,14 +468,13 @@ state
     return memo
   }, [])
 
-  const stackingMethod = stackItems ? stack : nostack
-
   // Get a new array of groupOrders holding the stacked items
-  const { height, groupHeights, groupTops } = stackingMethod(
+  const { height, groupHeights, groupTops } = stackAll(
     dimensionItems,
     groupOrders,
     lineHeight,
-    groups
+    groups,
```

Comment:
@Ilaiwi This is the error, The groups argument is not needed for stackAll as it now comes in with groupOrders